pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1782,6 +1785,80 @@ func TestPodMatchesEvent(t *testing.T) {
 	}
 }
 
+func TestMoveAllToActiveOrBackoffQueue_PreEnqueueChecks(t *testing.T) {
+	var podInfos []*framework.QueuedPodInfo
+	for i := 0; i < 5; i++ {
+		pInfo := newQueuedPodInfoForLookup(&v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{Name: fmt.Sprintf("p%d", i)},
+			Spec:       v1.PodSpec{Priority: pointer.Int32Ptr(int32(i))},
+		})
+		podInfos = append(podInfos, pInfo)
+	}
+
+	tests := []struct {
+		name             string
+		preEnqueueChecks []framework.PreEnqueueCheck
```

Comment:
I use mock checks here. Let me know if you'd like to switch to real PreEnqueueCheck.

Reply:
Ack, will do later.

Reply:
I didn't put UT in cache_test.go as cache.AddNode, cache.UpdateNode and cache.Clone have been covered properly. I added a test in eventhandlers_test.go, assuming the returned NodeInfo is correct. PTAL.